tests/zombienet.toml

```diff
@@ -3,6 +3,7 @@ chain = "rococo-local"
 
 [[relaychain.nodes]]
 name = "alice"
+rpc_port = 8833
```

Comment:
Shouldnt be included in this PR (out of scope)

Reply:
Removed.